Added Design for Developers

stjepanjurekovic opened the pull request:
Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Design for Developers

Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

Ki-er commented:
Hi, as this list is not maintained anymore, I have added it to my fork found here: https://github.com/KieranRobson/awesome-web-design/blob/master/README.md

stjepanjurekovic commented:
Thank you.

Best,

On Sat, May 7, 2022 at 10:42 PM Kieran ***@***.***> wrote:

> Hi, as this list is not maintained anymore, I have added it to my fork
> found here:
> https://github.com/KieranRobson/awesome-web-design/blob/master/README.md
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nicolesaidy/awesome-web-design/pull/33#issuecomment-1120286429>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMPENP7Q2VY2ZUJVC4LVI3ISVANCNFSM5U7VNZBQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***

stjepanjurekovic committed:
Added Web design playground, second edition

Hi,

Stjepan here from Manning. I thought this title might be a good match for your list. Thank you for considering it.

Best,
